cargo-shuttle/src/lib.rs

```diff
@@ -843,12 +843,9 @@ impl Shuttle {
             while let Ok(message) = rx.recv() {
                 match message {
                     Message::TextLine(line) => println!("{line}"),
-                    Message::CompilerMessage(message) => {
-                        if let Some(rendered) = message.message.rendered {
-                            println!("{rendered}");
-                        }
+                    message => {
+                        trace!("skipping cargo line: {message:?}")
```

Comment:
Is this to avoid the super verbose cargo messages we say during builds?

Reply:
Yes